operate/client/e2e-playwright/pages/Processes/Processes.ts

```diff
@@ -6,48 +6,23 @@
  * except in compliance with the Camunda License 1.0.
  */
 
-import {Page, Locator, expect} from '@playwright/test';
+import {Page, Locator} from '@playwright/test';
 import {convertToQueryString} from '../../utils/convertToQueryString';
 import {Paths} from 'modules/Routes';
 import {DeleteResourceModal} from '../components/DeleteResourceModal';
 import MigrationModal from '../components/MigrationModal';
 import MoveModificationModal from '../components/MoveModificationModal';
 import {Diagram} from '../components/Diagram';
-
-type OptionalFilter =
```

Comment:
I moved all filters panel related things to a new FiltersPanel class. I realized that the Processes class grew very big and it was hard to find things.